Server/Ebenezer/User.cpp

```diff
@@ -2412,6 +2412,12 @@ void CUser::Regene(char* pBuf, int magicid)
 				}
 				else return;
 			}
+			int16_t spawnX = 0, spawnZ = 0;	// The point where you will be warped to.
```

Comment:
This is overriding all of the previous blocks. I assume this was intended to be in an `else` block following them.